libraries/AP_HAL_ChibiOS/hwdef/revo-mini-sd/hwdef-bl.dat

```diff
@@ -0,0 +1,38 @@
+# hw definition file for processing by chibios_pins.py
+# for minimal F405 bootloader
```

Comment:
Please instead include the hwdef on which this is based, undef the pins you want to change and then put in the definitions you want